example/App.tsx

```diff
@@ -2,16 +2,18 @@ import * as React from 'react';
 import { Button, View } from 'react-native';
 
 import queue from '../src/Queue';
-import { Worker } from '../src/Worker';
+import { Worker, CANCEL, CancellablePromise } from '../src/Worker';
 
 export interface IAppProps {}
 
-export interface IAppState {}
+export interface IAppState {jobId?: string}
```

Comment:
The type should be { jobId: string | null }.
The current optional would infer that the jobId is string or undefined (not null).

Reply:
Nice